src/Tasks/system.design/stronglytypedresourcebuilder.cs

```diff
@@ -490,41 +485,60 @@ private static string TruncateAndFormatCommentStringForOutput(string commentStri
         [SuppressMessage("Microsoft.Globalization", "CA1303:DoNotPassLiteralsAsLocalizedParameters")]
         private static bool DefineResourceFetchingProperty(String propertyName, String resourceName, ResourceData data, CodeTypeDeclaration srClass, bool internalClass, bool useStatic)
         {
-            var prop = new CodeMemberProperty
-            {
-                Name = propertyName,
-                HasGet = true,
-                HasSet = false
-            };
+            CodeTypeReference valueType;
+            bool isString;
+            bool isStream;
 
-            Type type = data.Type;
-            if (type == null)
+            if (data.Type is Type type)
             {
-                return false;
-            }
+                // We have an actual object on which we can do type comparisons
+
+                if (type == typeof(MemoryStream))
+                {
+                    type = typeof(UnmanagedMemoryStream);
+                }
+
+                // Ensure type is internalally visible.  This is necessary to ensure
```

Comment:
Typo: internalally (also further down on comment)

Reply:
Existing, but I'd love to clean it up for 16.4.